debugger/src/main/java/org/codice/acdebugger/breakpoints/SecurityCheckInformation.java

```diff
@@ -168,30 +170,32 @@ private SecurityCheckInformation(
         Collections.emptyList());
     this.context = new ArrayList<>(context.size());
     this.local_i = local_i;
-    this.currentDomain = context.get(local_i);
-    this.currentBundle = debug.bundles().get(currentDomain);
+    this.currentDomainValue = context.get(local_i);
+    this.currentDomain = debug.locations().get(currentDomainValue);
     this.domains = new ArrayList<>(context.size());
     this.permission = permission;
     this.privilegedDomains = new HashSet<>(context.size() * 3 / 2);
     for (int i = 0; i < context.size(); i++) {
-      if (i == local_i) { // we know we don't have privileges since we failed here
```

Comment:
ℹ️ This was improperly skipping the current domain and it wasn't being added to the context. So it got moved a little bit further down.